tests/iterator/workdir.c

```diff
@@ -1170,7 +1170,7 @@ void test_iterator_workdir__pathlist_for_deeply_nested_item(void)
 
 		cl_git_pass(git_iterator_for_workdir(&i, g_repo, NULL, NULL, &i_opts));
 		expect_iterator_items(i, expected_len, expected, expected_len, expected);
-		cl_assert_equal_i(14, i->stat_calls);
+		cl_assert_in_range(7, i->stat_calls, 14);
```

Comment:
I think changes that have reduced the number of stat calls aren't in this PR. However, since I merely relaxed the assertions, tests will pass either way.